IMC_Python/Storing_Data_In_Python/Storing_Data_in_Python.md

```diff
@@ -0,0 +1,253 @@
+# Storing Data in Python
+
+When we think about Python, we can think of tons of applications for our potential code; we can create desktop applications, build websites, analyze data, and much more - the possibilities are endless. However, in order to do anything of that caliber with this programming language, you must start from the bottom and master the basics. One essential building block of Python is to learn to store pieces of data that will be useful to our program and, more importantly, decide how we will store this data. Lists, sets, and dictionaries are the most common ways to store data. In this blog post, I will show you how you can use them, why you would want to use them, and what features each data structure has to offer. 
+
+The first data structure I will talk about is probably the most commonly used in Python - lists. With this method, you can store variables of the same type in a specific order. The, you can access and modify them with an index. 
+
+## Lists
+
+**Initialization**
+
+*list= [element1, element2, …, elementn]*
+
+When you initialize a list, you must give it a name. If you choose to add elements to the list at this time (optional), they must be separated by commas and wrapped in square brackets, [].
+
+Ex: Take a list of integers called **numbers**. Let’s suppose you want it to contain numbers 0-4. This is how we would initialize the list: 
+
+<center>numbers = [0, 1, 2, 3, 4]</center>
+
+If we wanted **numbers** initialized to be empty, its declaration would look like this:
+
+<center>numbers = []</center>
+
+**Accessing an Element**
```

Comment:
It might help to increase the size of your bolded headers so they can be distinguished from the regular bolded words. Also, ensure there equal spacing between the last sentence and the next section. I noticed this before accessing an element and adding an element.